src/Raven.Client/Documents/Operations/Counters/GetCountersOperation.cs

```diff
@@ -120,7 +123,7 @@ private void PrepareRequestWithMultipleCounters(StringBuilder pathBuilder, HttpR
                         Operations = new List<CounterOperation>()
                     };
 
-                    foreach (var counter in _counters)
+                    foreach (var counter in uniqueNames)
```

Comment:
Do we have to worry about the order of operations here?

Reply:
What happened if the user provided the same counter twice?

Reply:
Is the user going to expect index matching on input and output?
What about the `null` here, should there be a match here?

Reply:
- no, the order stays the same
- if the user provided the same counter more than once, we will drop the duplicates from the request. I don't think it makes much sense to ask for same counter twice... 
- if we have a `null` counter name here, we will get a NRE here: https://github.com/ravendb/ravendb/blob/v4.2/src/Sparrow.Server/ByteString.cs#L1268

Reply:
I was actually asking about what we'll get on the *client* side.
Will we get the duplicate counter value twice as a result?
What happens when we have a null?

Also, I don't think that I saw protection on the server side for this?

Reply:
on the client side (before these changes) - if the user asked for a small batch with duplicates, he will get the counter value once as a result. 
If the user asked for a large batch with a duplicate counter name, he will get the counter value twice in the result. 
if the user had any `null` counter name in his request. it will fail (regardless to batch size)

And you're right, there's not much protection on the server side for this ..

Reply:
Similar problem in `GetDocumentsCommand` - if the user had any `null` ids in his request - it will fail (regardless to batch size). And duplicate ids are always removed in this command

Reply:
Let's add protection on the server side, and return null for such an instance.
For `GetDocumentsCommand` as well.

And if we match the client side behavior now of `GetDocumentsCommand`, it is good

Reply:
done